ansible/roles/status-checker/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+webserver_user: user
+webserver_directory: /var/user
```

Comment:
Why aren't those already `webserver` and `/var/webserver` in order not to have to overwrite them in the playbook?

Besides, I think it would be clearer to have an user called `checker` or something like that, `webserver` is not very explicit for the purpose of the webserver.